go.sum

```diff
@@ -0,0 +1,13 @@
+github.com/OneOfOne/xxhash v1.2.2/go.mod h1:HSdplMjZKSmBqAxg5vPj2TmRDmfkzw+cTzAElWljhcU=
+github.com/cespare/xxhash v1.1.0 h1:a6HrQnmkObjyL+Gs60czilIUGqrzKutQD6XZog3p+ko=
+github.com/cespare/xxhash v1.1.0/go.mod h1:XrSqR1VqqWfGrhpAt58auRo0WTKS1nRRg3ghfAqPWnc=
+github.com/coocood/freecache v1.1.0 h1:ENiHOsWdj1BrrlPwblhbn4GdAsMymK3pZORJ+bJGAjA=
+github.com/coocood/freecache v1.1.0/go.mod h1:ePwxCDzOYvARfHdr1pByNct1at3CoKnsipOHwKlNbzI=
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
```

Comment:
We don't have this dependency in go.mod, that's strange that it's here 🤔

Reply:
Those are dependencies of our testing/benchmark dependencies.

```bash
$ go mod why github.com/OneOfOne/xxhash
# github.com/OneOfOne/xxhash
github.com/allegro/bigcache/caches_bench
github.com/coocood/freecache
github.com/cespare/xxhash
github.com/cespare/xxhash.test
github.com/OneOfOne/xxhash
```